src/structure.ts

```diff
@@ -81,44 +81,48 @@ const isCommand = (value: any, warn = false): value is Command => {
     return true;
 };
 
+type AccType = { [key: string]: Command };
+
 /**
  * Validate that a set of values are {@link Command} objects
  */
-export const validateCommands = <Ctx extends Context = Context, Req extends Request = Request, Sentry extends Toucan | undefined = undefined>(cmds: any[], warn = false) =>
-    cmds.reduce((acc, cmd) => {
-        if (!isCommand(cmd, warn)) return acc;
+export const validateCommands = <Ctx extends Context = Context, Req extends Request = Request, Sentry extends Toucan | undefined = undefined>(cmds: unknown[], warn = false) =>
+cmds.reduce((acc: AccType, cmd: unknown) => {
+    if (!isCommand(cmd as Command, warn)) return acc;
 
-        // Check the command doesn't already exist
-        if (acc[cmd.name]) {
-            if (warn)
-                console.warn(`Command ${cmd.name} already exists`);
-            return acc;
-        }
+    const command = cmd as Command;
```

Comment:
This line shouldn't be needed, the isCommand call above tells typescript that cmd is of type Command